README.md

```diff
@@ -33,6 +33,8 @@
    ```js
    export default defineNuxtConfig({
      modules: ['@nuxtjs/turnstile'],
+     // Pass a path to a file containing the secret key by passing it in the module options like in the line below
+     // modules: [['@nuxtjs/turnstile', { secretKeyPath: 'turnstile.secret' }]],
      
      turnstile: {
        siteKey: '<your-site-key>',
```

Comment:
The `turnstile` configuration is already defined. Why add it a second, untyped time under modules?